lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1280,7 +1281,7 @@ abstract class DefaultTextEditingStrategy with CompositionAwareMixin implements
       activeDomElement.setAttribute('type', 'password');
     }
 
-    if (config.inputType == EngineInputType.none) {
+    if (config.inputType == EngineInputType.none || config.inputType == EngineInputType.multilineNone) {
       activeDomElement.setAttribute('inputmode', 'none');
     }
```

Comment:
Small suggestion to simplify this a bit:
```suggestion
    if (config.inputType.inputmodeAttribute == 'none') {
      activeDomElement.setAttribute('inputmode', 'none');
    }
```